src/documentation/pages/components/icon/examples/_brand_complimentary.vue

```diff
@@ -0,0 +1,52 @@
+<template lang="pug">
+<ComponentExample title="Brand complimentary" titleSize="h4" padding="-p--0" id="brand_complimentary" :tabs="exampleTabs">
+  p.-text(slot="example-description")
+    | More icon colors that can be used to compliment the primary brand palette.
+  .chi-grid.-no-gutter(slot="example")
```

Comment:
Can we use this?

```suggestion
  <Wrapper slot="example">
```

Within the Wrapper we may be able to use `.chi-grid.-no-gutter` and other classes.